datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/types/MutableType.java

```diff
@@ -24,4 +24,14 @@
      * @param context the {@link QueryContext} corresponding to the request.
      */
     T update(@Nonnull final String urn, @Nonnull final I input, @Nonnull final QueryContext context) throws Exception;
+    
+   /**
+    * Update many entities
+    *
+    * @param input input type
+    * @param context the {@link QueryContext} corresponding to the request.
+    */
+   default <T> T batchUpdate(@Nonnull final I[] input, @Nonnull final QueryContext context) throws Exception {
+    return null;
```

Comment:
Should we return null here, or simply throw an UnsupportedOperationException if the type does not implement the function?

Reply:
I've changed the default behaviour to throw an UnsupportedOperationException